src/widget/wknob.cpp

```diff
@@ -19,14 +19,15 @@
 #include <QMouseEvent>
 #include <QWheelEvent>
 
-#include "util/time.h"
+#include "util/duration.h"
 #include "widget/wknob.h"
 
 WKnob::WKnob(QWidget* pParent)
         : WDisplay(pParent),
-          m_guiTickTimer(this) {
-    connect(&m_guiTickTimer, SIGNAL(timeout()),
-            this, SLOT(guiTick()));
+          m_renderTimer(mixxx::Duration::fromMillis(20),
+                        mixxx::Duration::fromSeconds(1)) {
+    connect(&m_renderTimer, SIGNAL(update()),
```

Comment:
How about moving this to the WidgetRenderTimer constructor by passing a pointer to the parent widget as an argument?

Reply:
I dunno, I think that's up the user of the utility class -- for example using update vs repaint could be a per-widget decision. Or maybe it's not a QWidget at all and driving some QML object?

Reply:
(I also don't like classes in `src/util` to depend on QtWidgets)

Reply:
Okay